src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
```

Comment:
Remove everything except box-sizing from this selector.
It is a bad practice to use `*` selector for styling